### PR TITLE
Feature/12988-FSS-UI-Year-/-Week-attribute-to-display-in-descending-order-on-the-UI

### PR DESCRIPTION
**Pull request by prasad14878**

### CODE REVIEW

File: Tests/UnitTests/fss-search.component.spec.ts [hunk 1]
```diff
@@ -140,10 +141,10 @@ export const attributeSearchFilterMockData: any =
          "expanded": true
       },
       {
-         "title": "weeknumber",
+         "title": "Year / Week",
```
Comment (crdandoukho) [inline Tests/UnitTests/fss-search.component.spec.ts:144]: Please check if `attributeSearchFilterMockData` is actually used. If not, please delete it.

File: src/assets/config/appconfig.json [hunk 2]
```diff
@@ -17,27 +17,31 @@
     "copyright": "@ Crown copyright 2021 UK Hydrographic office",
     "fssTitle": "File Share Service",
     "displayPopularSearch": true,
-    "displaySimplifiedSearchLink": true,
+    "displaySimplifiedSearchLink": false,
```
[the remark this answers is not in the document]
Comment (prasad14878) [reply src/assets/config/appconfig.json]: Resolved

File: src/app/features/fss-search/fss-search.component.ts [hunk 3]
```diff
@@ -181,7 +181,7 @@ export class FssSearchComponent implements OnInit {
   onApplyFilterButtonClicked(filterItem: FilterGroup[]){
     if (!this.fileShareApiService.isTokenExpired()) {
       var filterExpression = this.fssSearchFilterService.getFilterExpressionForApplyFilter(filterItem);
-      var applyFilter_FilterExpression = this.MainQueryFilterExpression.concat(" AND ").concat("(" + filterExpression + ")");
+      var applyFilter_FilterExpression = filterExpression !== "" ? this.MainQueryFilterExpression.concat(" AND ").concat("(" + filterExpression + ")") : this.MainQueryFilterExpression;
```
[the remark this answers is not in the document]
Comment (prasad14878) [reply src/app/features/fss-search/fss-search.component.ts]: Resolved

File: Tests/UnitTests/fss-search.component.spec.ts [hunk 4]
```diff
@@ -104,9 +105,9 @@ export const inputSearchResultMockData: any =
          ]
       },
       {
-         "key": "weeknumber",
+         "key": "Year / Week",
```
[the remark this answers is not in the document]
Comment (prasad14878) [reply Tests/UnitTests/fss-search.component.spec.ts]: Resolved

File: src/app/features/fss-search/fss-search.component.ts [hunk 5]
```diff
@@ -316,19 +316,22 @@ export class FssSearchComponent implements OnInit {
...
             expanded: true
           });
         }
       }
     }
   }
 
-  getAttributesValues(attributeValues: Array<any> = [],attributeSortType: any) {
-    if(attributeSortType==="numeric"){
+  getAttributesValues(attributeValues: Array<any> = [], attributeSortType: any, sortOrder: any) {
```
[the remark this answers is not in the document]
Comment (prasad14878) [reply src/app/features/fss-search/fss-search.component.ts:327]: Resolved

[the remark this answers is not in the document]
Comment (prasad14878) [reply src/app/features/fss-search/fss-search.component.ts:327]: Resolved

File: Tests/UnitTests/fss-search.component.spec.ts (hunk 1 above)
Comment (prasad14878) [reply Tests/UnitTests/fss-search.component.spec.ts]: Resolved

**Review by crdandoukho — APPROVED** [review]
Thanks Prasad for your changes.

Looks good.

I will delay merging until after show and tell.

**Review by bhalcha13813 — APPROVED** [review]
LGTM

**Pull request merged**